client/mds3/skewer.render.js

```diff
@@ -270,6 +270,10 @@ export function skewer_make(tk, block) {
 		.on('click', async (event, d) => {
 			click_variant(d, tk, block, event.target.getBoundingClientRect(), event.target)
 		})
+		.on('transitionend', () => {
+			console.log(274)
```

Comment:
delete

Reply:
Sure thing. Done in [3f8c62a](https://github.com/stjude/proteinpaint/pull/2132/commits/3f8c62a0987115d8df5882dd0456bee16f52a379)